command-expansion-server/test/sequence-generation.spec.ts

```diff
@@ -0,0 +1,242 @@
+import { gql, GraphQLClient } from 'graphql-request';
```

Comment:
This test seems like it would do well as part of our `e2eTest` suite that runs on every PR.

Reply:
Would love to add this in. Not sure how to and won't hold up this PR for it.